evaluate_library_access_from_output_tsv/facilitate_going_through_dois_manually.R

```diff
@@ -0,0 +1,74 @@
+#### Settings ####
+
+manual_tsv_location <- file.path(
+  'evaluate_library_access_from_output_tsv',
+  'stratefied_sample_of_dois_and_manual_full_text_check_results.tsv'
+)
+
+#### Open the tsv ####
+
+dataset_to_go_through <- read.table(
+  manual_tsv_location,
+  sep = '\t',
+  na.strings = '',
+  header = TRUE,
+  colClasses = 'character'
+)
+# View(dataset_to_go_through)
+
+#### Facilitate going through the rows that haven't been filled in ####
+
+for (row_number in 
+  which(
+    is.na(dataset_to_go_through$full_text_indicator_manual)
+  )
+) {
+  doi_for_row <- dataset_to_go_through$doi[row_number]
+  
+  url_to_visit <- paste0(
+    'https://doi.org/',
+    doi_for_row
+  )
+  
+  message('Opening URL "', url_to_visit, '"...')
+  
+  utils::browseURL(url_to_visit)
```

Comment:
This curator application is a cool concept. I'd be worried that it'll be difficult to jump around between DOIs... but if it helps you, then use this app.

I'm not going to review it extensively because the actual output dataset is the important one, it's up to you as the curator to fill it in however you find best. So feel free to do this if it helps.